src/askscript-vscode/vsc-extension-quickstart.md

```diff
@@ -2,28 +2,28 @@
 
 ## What's in the folder
 
-* This folder contains all of the files necessary for your extension.
```

Comment:
Any good reason for these formatting changes? I would like to avoid you changing from * to - and then somebody else changing back from - to *, so if reasonable, please revert changes in this file.

Reply:
Yes, it's only a reformatting issue. I executed `npm run prettier`, and inadvertently reformatted reformatted files that were not to intended to be.

**Previous formatting has been restored.**